app/assets/stylesheets/modules/sitter.scss

```diff
@@ -41,6 +44,50 @@
       padding-top: 2.6em;
     }
   }
+  }
+}
+
+//SINGLE
+#sitter-container {
+  font-family: $sitter-font-type;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+
+  .hr {
+    opacity: 0.3;
+  }
+
+  .sitter-profile {
+    padding-top: 3em;
+    margin: 0;
+
+    .small-6 {
+      padding: 0;
+      margin:0;
+
+      img {
+        @include image-style;
+      }
+
+      .row-name {
+        text-align: center;
+      }
+    }
+
+      .headline {
```

Comment:
Line should be indented 4 spaces, but was indented 6 spaces<br>Selector should have depth of applicability no greater than 2, but was 3